cse/SearchEngine.py

```diff
@@ -99,33 +94,40 @@ def index(self):
         if self.__indexLoaded:
             self.releaseIndex()
 
-        FileIndexer(self.__directory, self.__prep).index()
+        FileIndexer(
+            self.__directory,
+            self.__commentsFilename,
+            self.__articleFilename,
+            self.__authorFilename,
+            self.__prep
+        ).index()
 
 
     def close(self):
         self.releaseIndex()
 
 
-    def search(self, query, topK=10):
+    def search(self, query, idsOnly=False, topK=10):
         if not self.__indexLoaded:
             print("Index was not loaded!")
             return []
 
         results = []
+        print()
```

Comment:
?

Reply:
prints a newline to separate the output of multiple calls to `search()`